### DISCUSSION TITLE
ffmpeg path

### DISCUSSION THREAD

**Discussion by heinetz** [cwilby/nextcloud-workflow-media-converter#393] [Ideas]
Hi,
my nc is on shared hosting without ffmpeg. I just got preview images for videos working with ffmpeg static build. it works because of following option in config file:

'preview_ffmpeg_path' => '/path/to/ffmpeg-git-20230915-amd64-static/ffmpeg',

Is there such an option for ur media converter ?

best,
heinetz

**Comment by csss1234** [comment]
@heinetz did you ever figure it out?
